lib/utils.py

```diff
@@ -68,6 +69,11 @@ def cleanup(varg: Dict[str, Any]) -> None:
             if entry.name.startswith("js_") and time.time() - sb.st_mtime > 604800:
                 cleandir(entry.name, verbose=varg["verbose"])
 
+    # if our BEARER_TOKEN_FILE ends with our pid, then we copied it and should
+    # clean it up...
+    if token_mods.is_copied_token(os.environ["BEARER_TOKEN_FILE"]):
```

Comment:
Can we also add a verbose clause here, where if `varg["verbose"] > 0`, we print that we cleaned up that file?

Reply:
Done.

Reply:
Did you push that commit to Github?  The PR looks the same as before.